packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -220,7 +221,7 @@ function getCommonEventData<T extends keyof DocumentEventMap>(eventConfig: UIEve
     autoLoggingID,
     metadata,
     value,
-    pageURI: window.location.href,
+    pageURI: getCurrMainPageUrl(),
```

Comment:
This is safe to land as is?  Or should we make sure we cover all cases before relying on the new tracking?

Reply:
Ok, I updated the code to cover all possible cased for change of the URL. Had to add a `window.history` interception.
Tested it on internal tools and works correctly. 
See the equivalent diff on www, all references to the `pageURI` is already fixed.